Lib/tkinter/test/test_ttk/test_extensions.py

```diff
@@ -291,6 +291,32 @@ def cb_test(item):
 
         optmenu.destroy()
 
+    def test_unique_radiobuttons(self):
+        # check that radiobuttons are unique across instances (bpo25684)
+        items = ('a', 'b', 'c')
+        default = 'a'
+        optmenu = ttk.OptionMenu(self.root, self.textvar, default, *items)
+        textvar2 = tkinter.StringVar(self.root)
+        optmenu2 = ttk.OptionMenu(self.root, textvar2, default, *items)
+        optmenu.pack()
+        optmenu.wait_visibility()
+        optmenu2.pack()
+        optmenu2.wait_visibility()
+        optmenu['menu'].invoke(1)
+        optmenu2['menu'].invoke(2)
+        optmenu_radiobutton_name = optmenu['menu'].entrycget(0, 'variable')
+        optmenu2_radiobutton_name = optmenu2['menu'].entrycget(0, 'variable')
+        self.assertNotEqual(optmenu_radiobutton_name,
+                            optmenu2_radiobutton_name)
+        self.assertEqual(self.root.tk.globalgetvar(optmenu_radiobutton_name),
+                         items[1])
+        self.assertEqual(self.root.tk.globalgetvar(optmenu2_radiobutton_name),
+                         items[2])
```

Comment:
I had a lot of trouble with this when I was trying to create the test.  I had even asked for help on core_workflow.  But, after looking through the other tests in this suite, I saw that `globalgetvar` gave me what I wanted.  I ran the test over the old code and it failed (which was good - failed on the old and passed on the new) and I also looked at the values in debug and this was exactly what I needed.  I don't know if there was any other way to get that information.